tgui/packages/tgui/interfaces/HairPicker.tsx

```diff
@@ -0,0 +1,226 @@
+import { hexToHsva, HsvaColor, hsvaToHex } from 'common/color';
+import { BooleanLike } from 'common/react';
+import { createRef, useState } from 'react';
+
+import { useBackend } from '../backend';
+import {
+  Box,
+  Button,
+  ColorBox,
+  DmIcon,
+  Dropdown,
+  Modal,
+  Section,
+  Stack,
+  Tooltip,
+} from '../components';
+import { Window } from '../layouts';
+import { ColorSelector } from './ColorPickerModal';
+
+type HairPickerData = {
+  hair_icon: string;
+  hair_styles: { name: string; icon: string }[];
+  hair_style: string;
+  hair_color: string;
+
+  facial_hair_icon: string;
+  facial_hair_styles: { name: string; icon: string }[];
+  facial_hair_style: string;
+  facial_hair_color: string;
+
+  gradient_available: BooleanLike;
+  gradient_styles: string[];
+  gradient_style: string;
+  gradient_color: string;
+};
+
+export const HairPicker = () => {
+  const { act, data } = useBackend<HairPickerData>();
+
+  const {
+    hair_icon,
+    hair_style,
+    hair_styles,
+    hair_color,
+    facial_hair_icon,
+    facial_hair_style,
+    facial_hair_styles,
+    facial_hair_color,
+    gradient_available,
+    gradient_style,
+    gradient_styles,
+    gradient_color,
+  } = data;
+
+  const [colorPicker, setColorPicker] = useState<
+    'hair' | 'facial_hair' | 'gradient' | false
+  >(false);
+
+  let height = 340;
+  if (facial_hair_styles.length > 1) {
+    height = height + 300;
```

Comment:
```suggestion
    height = height + 310;
```
![image](https://github.com/user-attachments/assets/62567f21-9898-40f9-ac87-9027ff622bac)
instead of
![image](https://github.com/user-attachments/assets/88758dd6-f284-40f1-b983-eb1d545fa4e5)